include/z64skin.h

```diff
@@ -0,0 +1,85 @@
+#ifndef Z64_SKIN_H
+#define Z64_SKIN_H
+
+#include "z64animation.h"
+
+/**
+ * Holds a compact version of a vertex used in the Skin system
+ * It is used to initialise the Vtx used by an animated limb
+ */
+typedef struct {
+    /* 0x00 */ u16 vtxIndex;
+    /* 0x02 */ s16 u; // u and v are texture coordinates
+    /* 0x04 */ s16 v;
+    /* 0x06 */ s8 x;
+    /* 0x07 */ s8 y;
+    /* 0x08 */ s8 z;
+    /* 0x09 */ u8 alpha;
+} SkinVtx; // size = 0xA
+
+/**
+ * Describes a position displacement and a scale to be applied to a limb at index `limbIndex`
+ */
+typedef struct {
+    /* 0x00 */ u8 limbIndex; // the limb to apply this modifications
+    /* 0x02 */ s16 x;
+    /* 0x04 */ s16 y;
+    /* 0x06 */ s16 z;
+    /* 0x08 */ u8 scale;
+} SkinLimbModif; // size = 0xA
+
+typedef struct {
+    /* 0x00 */ u16 skinVerticesCount; // count of skinVertices
+    /* 0x02 */ u16 limbsModificationsCount; // count of limbsModifications
+    /* 0x04 */ u16 unk_4; // index of limbsModifications?
+    /* 0x08 */ SkinVtx* skinVertices;
+    /* 0x0C */ SkinLimbModif* limbsModifications;
+} Struct_800A598C; // size = 0x10
+
+typedef struct {
+    /* 0x00 */ u16 vtxCount;
+    /* 0x02 */ u16 unk_2; // count of unk_4
+    /* 0x04 */ Struct_800A598C* unk_4;
+    /* 0x08 */ Gfx* dlist;
+} SkinAnimatedLimbData; // size = 0xC
+
+// ZAPD compatibility typedefs
```

Comment:
Could you add a TODO to remove these when ZAPD is compatible?

Reply:
```suggestion
// ZAPD compatibility typedefs
// TODO: Remove when ZAPD adds support for them
```